sqrl-planner/src/main/java/com/datasqrl/engine/server/GenericJavaServerEngine.java

```diff
@@ -65,26 +59,17 @@ public EnginePhysicalPlan plan(com.datasqrl.planner.dag.plan.ServerStagePlan ser
 
   @SneakyThrows
   private String serverConfig() {
-    var mergedConfig = mergeConfigs(readDefaultConfig(), engineConfig.getConfig());
-    return objectMapper.writer(new PrettyPrinter()).writeValueAsString(mergedConfig);
+    var mergedConfig = ServerConfigUtil.mergeConfigs(readDefaultConfig(), engineConfig.getConfig());
+    return mapper.writer(new PrettyPrinter()).writeValueAsString(mergedConfig);
```

Comment:
Again, this changes the common `mapper`, I think a class level copy with the `PrettyPrinter` applied is the way to go.